core/src/mindustry/ui/dialogs/AboutDialog.java

```diff
@@ -1,17 +1,21 @@
 package mindustry.ui.dialogs;
 
 import arc.*;
+import arc.func.Cons;
```

Comment:
This doesn't follow style guidelines, you are not using wildcard imports and this very much looks like a redundant import.